imap/imap.go

```diff
@@ -0,0 +1,365 @@
+package imap
+
+// Functionality taken from https://github.com/jprobinson/eazye
+
+import (
+	"bytes"
+	"crypto/tls"
+	"fmt"
+	"strconv"
+	"time"
+
+	log "github.com/gophish/gophish/logger"
+	"github.com/gophish/gophish/models"
+	"github.com/jordan-wright/email"
+	"github.com/mxk/go-imap/imap"
+)
+
+// Client interface for IMAP interactions
+type Client interface {
+	Close(expunge bool) (cmd *imap.Command, err error)
+	Login(username, password string) (cmd *imap.Command, err error)
+	Logout(timeout time.Duration) (cmd *imap.Command, err error)
+	Select(mbox string, readonly bool) (cmd *imap.Command, err error)
+	UIDFetch(seq *imap.SeqSet, items ...string) (cmd *imap.Command, err error)
+	UIDSearch(spec ...imap.Field) (cmd *imap.Command, err error)
+	UIDStore(seq *imap.SeqSet, item string, value imap.Field) (cmd *imap.Command, err error)
+}
+
+// UIDStore stores UIDs
```

Comment:
Looks like this bit of commented code slipped into the commit.

Reply:
Done